CMakeLists.txt

```diff
@@ -381,12 +381,23 @@ macro(nebula_add_test)
 endmacro()
 
 macro(nebula_build_after_base obj_name)
-    add_dependencies(${obj_name} base_obj)
 endmacro()
```

Comment:
delete the unused macro `nebula_build_after_base`.

Reply:
My bad